docs/source/index.adoc

```diff
@@ -0,0 +1,31 @@
+[[minishift-index]]
+= Minishift
```

Comment:
-1, the landing page is in overview.adoc. By creating this file you are destroying the entry point into the site for Middleman? How do you preview the site?